consensus/src/epoch_manager.rs

```diff
@@ -947,6 +953,9 @@ impl EpochManager {
         peer_id: AccountAddress,
         event: &UnverifiedEvent,
     ) -> anyhow::Result<()> {
+        if self.recovery_mode {
+            return Ok(());
```

Comment:
it looks better to me to inline this with the quorum_store_enabled condition like this
```
  if self.quorum_store_enabled || self.recovery_mode
```

Reply:
Done